Source/Scene/Model.js

```diff
@@ -3468,6 +3469,10 @@ define([
                     mesh : runtimeMeshesByName[mesh.name]
                 };
 
+                if (defined(model._pickInfo)) {
+                    owner = combine(owner, model._pickInfo);
+                }
```

Comment:
I'm starting to rethink this. If the custom pick object exists then it should it be used directly, not combined with `owner`. I doubt anyone picking a tile needs to know what gltf node/mesh are picked.